lib/Element.js

```diff
@@ -181,23 +164,63 @@ function Element(document, $elm) {
     }
   });
 
+  function radioButtonChecked(value) {
+    const oldValue = (getAttribute("checked") === "checked");
+    uncheckRadioButtons();
+
+    if (value) {
+      setAttribute("checked", "checked");
+      if (!oldValue) {
+        dispatchEvent(new CustomEvent("change"));
```

Comment:
is it possible to do `dispatchEvent(new Event("change"))` since it's a native event?